src/components/modal/modal.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { CloseModal, ModalElement, ModalHeader, ModalWrapper } from "./modal.styled";
+
+interface IModalElement {
+    closeModal: () => void;
+    cleanModalData: () => void;
+    showModal: boolean;
+    children: React.ReactNode;
+};
+
+
+
+export const Modal: React.FC<IModalElement> = (props) => {
+    if (!props.showModal) {
+        return null;
+    };
+
+    return (
+        <ModalWrapper>
```

Comment:
poderia fazer apenas isso:

```suggestion
    return (
      <>
       {props.showModal && (
          <ModalWrapper>
```
so teria que ajustar a identacao do return